sdk/src/Core/Amazon.Runtime/Credentials/ImmutableCredentials.cs

```diff
@@ -50,7 +50,7 @@ public class ImmutableCredentials
         /// <summary>
         /// Gets the AccountId property for the current credentials.
         /// </summary>
-        public string AccountId { get; private set; }
+        public string AccountId { get; protected set; }
```

Comment:
I think this was done for `SSOImmutableCredentials` but that change doesn't need seem necessary. Since this type is supposed to be immutable I really don't want this change allowing a potential subclass to mutated the object.